docs/_includes/projects/index-typescript/index.ts

```diff
@@ -0,0 +1 @@
+import './custom-greeting.ts';
```

Comment:
because stackblitz, consistency, & clarity... 

- StackBlitz requires index.ts/js & errors if I import modules from html
- I want to show the user clear chunks of code rather than large files
- with multiple elements in an example, writing them all in index.* becomes clunky so I have separate files for each element - for consistency, I do this even if there is only 1 element

Reply:
I agree with separate files for elements in most cases (trying to show only how two elements can compose might be easier with two very small elements in one file).

The stackblitz limitations should be gone with the `lit-html` project template, which will hopefully be usable tomorrow. We can think about migrating to that later...